vignettes/meetupr.Rmd

```diff
@@ -0,0 +1,223 @@
+---
+title: "Setting up and Getting Started with meetupr"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Setting up and Getting Started with meetupr}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r}
+#| include: false
+#| label: setup
+
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  eval = TRUE
+)
+
+vcr::setup_knitr(prefix = "meetupr-")
+meetupr:::mock_if_no_auth()
+Sys.setenv(MEETUPR_DEBUG = "")
+```
+
+```{r}
+#| label: load
+library(meetupr)
+```
+
+## Quick Setup Check
+
+Start by running `meetup_sitrep()` to set up authentication or check your authentication status:
+
+```{r}
+#| label: sitrep
+#| cassette: true
+
+meetup_sitrep()
+```
+
+This function will show you:  
```

Comment:
No, this function _launched_ the authentication. Should it stop doing that and instead give the TODO to the user?

Pro: it'd be closer to what other sitrep functions do, i.e. only report.
Con: one click more for user.

Reply:
I think it shouldnt launch it. Had to redo some logic for that to work, bit sitrep will no longer initiate auth if its not present, only display information